processor/k8sattributesprocessor/doc.go

```diff
@@ -94,7 +94,28 @@
 //
 // Config
 //
-// TODO: example config.
+//      k8sattributes:
+//      k8sattributes/2:
+//        auth_type: "serviceAccount"
+//        passthrough: false
+//        filter:
+//          node_from_env_var: KUBE_NODE_NAME
+//
+//        extract:
+//          metadata:
+//            - k8s.pod.name
+//            - k8s.pod.uid
+//            - k8s.deployment.name
+//            - k8s.cluster.name
+//            - k8s.namespace.name
+//            - k8s.node.name
+//            - k8s.pod.start_time
+//
+//        pod_association:
+//         - from: resource_attribute
+//           name: k8s.pod.ip
+//         - from: resource_attribute
+//           name: k8s.pod.uid
```

Comment:
I think it would be good to include connection association in the example, since it's fairly commonly used for tracing
```suggestion
//           name: k8s.pod.uid
//         - from: connection
//           name: ip
```